src/__tests__/Grid.Col.test.js

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-console */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { shallow } from 'enzyme';
+
+import { Col } from '../Grid';
+
+const MockComponent = props => <div css={props.styling} />;
```

Comment:
I'm going to propose a few things throughout this file, which relate to a change here.

We could have `MockComponent` accept a `className` prop that gets passed directly onto a `div`, ie.
```js
const MockComponent = ({ className }) => <div className={className} />;
```

This will allow the component to be used with a `css` prop, and gets us closer to how these pieces fit together in practice.

Reply:
I'm a bit confused about what you're requesting here. Are you saying that we need to demonstrate in the unit tests that the MockComponent can accept `css` and `classname` props? ie. are you requesting better / more extensive test coverage, rather than a functionality change?

Reply:
In general, I think the `css` and `className` props are confusing enough without introducing a third name, `styling`, into the mix ;)

Since `MockComponent` is only used in one place, and we're not interrogating it in any way... I'll add a comment where I think we could get rid of it entirely. Ignore this one.